lib/cdo/delete_accounts_helper.rb

```diff
@@ -79,32 +79,34 @@ def clean_survey_responses(user_id)
 
   # Remove all user generated content associated with any PD the user has been through, as well as
   # all PII associated with any PD records.
-  # @param [Integer] The ID of the user to clean the PD content.
-  def clean_and_destroy_pd_content(user_id)
-    PeerReview.where(reviewer_id: user_id).each(&:clear_data)
+  # @param [User] user being purged
+  def clean_and_destroy_pd_content(user)
+    user.workshops_as_facilitator = []
```

Comment:
No `save` or `update` here because the user model gets saved at the end of the `purge_user` operation.

https://github.com/code-dot-org/code-dot-org/blob/3e4d43f69f1045a782d9e454940db78a8005738d/lib/cdo/delete_accounts_helper.rb#L255